### ISSUE TITLE
State not saved on phone entering sleep mode

### ISSUE THREAD

**Issue by CliffracerX** [shawntan/minicraft-android#4]
Will reset whole game.

Slight issue, hopefully someone can fix it.

**Comment by shawntan** [comment]
Yeah, it was one of the things I was working on before life got in the way. Heh.

I might get back round to fiddling with it when I have the time.

Glad to see there's still _some_ interest, thanks!

**Comment by CliffracerX** [comment]
I enjoyed playing Minicraft on the PC, loved messing with it on my kindle.

BTW, I did add some stuff on my local version, never figured out the saving
though.  :P

On Fri, Feb 14, 2014 at 9:19 PM, Shawn Tan notifications@github.com wrote:

> Yeah, it was one of the things I was working on before life got in the
> way. Heh.
> 
> I might get back round to fiddling with it when I have the time.
> 
> Glad to see there's still _some_ interest, thanks!
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/shawntan/minicraft-android/issues/4#issuecomment-35144980
> .

## 

_-may teh codez be werff u_
